docs/manual/node-steps/datadog.md

```diff
@@ -0,0 +1,68 @@
+# Datadog Node Steps (Enterprise)
+
+## Getting Started
+
+Before we begin using the DataDog plugins, we are going to add the APP and API keys. We have two different ways of doing this. We can either add it to the framework or to the specific project. 
+
+### Adding to the Framework
+
+In the `etc/framework.properties` file, add the following lines of code, depending on which job step plugin you are using. 
+
+#### Mute Host
+```bash
+framework.plugin.WorkflowNodeStep.datadog-mute-hosts.api_key=value
```

Comment:
@ltamaster Do we really have to have separate entries for every action?  Why can't we set a single datadog api_key and app_key value that apply to all of them?

Reply:
I believe these configs will work for the Step plugins:

``` properties
datadog.api-key-storage-path=path..
datadog.app-key-storage-path=path...
```